.github/actions/on_device_tests/action.yaml

```diff
@@ -79,11 +78,11 @@ runs:
           ${DIMENSION:+"--dimension" "$DIMENSION"} \
           ${USE_SHARDING:+"--unittest_shard_index" "${{ matrix.shard }}"} \
           ${ON_DEVICE_TEST_ATTEMPTS:+"--test_attempts" "$ON_DEVICE_TEST_ATTEMPTS"} \
-          --archive_path ${GCS_PATH}/artifacts.tar \
-          --gcs_xml_result_path=${GCS_RESULTS}
+          --archive_path "${GCS_PATH}/artifacts.tar" \
+          --gcs_xml_result_path "${GCS_PATH}/results/${GITHUB_RUN_ID}/"
```

Comment:
The arg in python script was renamed to `gcs_rseult_path` - are those intentionally different ?

Reply:
Done